src/v2/wallet.js

```diff
@@ -1030,21 +1030,19 @@ Wallet.prototype.prebuildTransaction = function(params, callback) {
     if (params.reqId) {
       this.bitgo._reqId = params.reqId;
     }
-    let response = yield this.bitgo.post(this.baseCoin.url('/wallet/' + this._wallet.id + '/tx/build'))
-    .send(whitelistedParams)
-    .result();
-
-    debug('postprocessing transaction prebuild: %O', response);
-
-    // postProcess prebuild may make more requests
-    if (params.reqId) {
-      response._reqId = params.reqId;
-    }
-    response = yield this.baseCoin.postProcessPrebuild(response);
```

Comment:
Why aren't we calling `postProcessPrebuild()` anymore?

Reply:
postProcessPrebuild does 2 things:
1) gets the chainhead blockheight,---- i moved earlier so it can be done in parallel
2) sets transaction.locktime to 1 + blockheight --- i just copied that over
I didn't want to call it because i dont want it to go get the chainhead again

Reply:
update: moved the transaction.locktime stuff to postprocessprebuild. 
removed blockheight query from postprocessprebuild, and instead let user pass it in as a param

Reply:
update 2: keep the original postprocessprebuild, but allow user to pass in blockHeight as a param in the prebuild, so its no longer a breaking change